### ISSUE TITLE
Do we now need java 1.8 to compile?

### ISSUE THREAD

**Issue by haroon-sheikh** [fitnesse/fitnesse-git-plugin#1]
Fitnesse compiles fine with 1.7 but plugin requires 1.8

```bash
Caused by: java.lang.UnsupportedClassVersionError: fitnesse/wiki/fs/GitFileVersionsController : Unsupported major.minor version 52.0
```

I'm using this plugin with maven, and its the only thing that complains about compiling with 1.7.

**Comment by tayloryork** [comment]
Agreed, why does it need to have 1.8?

**Comment by amolenaar** [comment]
This is probably the default (on my machine ;) ). I'll correct it.

**Comment by amolenaar** [comment]
Released version 1.1.1. Compiled with Java 7. Requires FitNesse 20160618.
